src/validation/assertions.ts

```diff
@@ -18,6 +18,12 @@ export function assertIsLessThan(value: any, high: any, errorMessage: string): v
   }
 }
 
+export function assertIsLessOrEqual(value: any, high: any, errorMessage: string): void {
```

Comment:
```suggestion
export function assertIsLessOrEqual(value: number | string, high: number | string, errorMessage: string): void {
```

Assuming the parameters are only possibly numbers the `| string` part won't be necessary, but Javascript...